src/deterministic_secure_broadcast.rs

```diff
@@ -417,7 +420,11 @@ impl<A: SecureBroadcastAlgorithm> SecureBroadcastProc<A> {
     }
 
     fn quorum(&self, n: usize) -> bool {
-        n * 3 >= self.peers.len() * 2
+        // 2/3.
+        // n * 3 >= self.peers.len() * 2
+
+        // simple majority.
+        n * 4 > self.peers.len() * 2
```

Comment:
doesn't work unfortunately,  we need > 2/3rds of the network to form quorum

Reply:
This is something we should discuss in slack or next hangout.

Reply:
Yes, if we use n*n and not gossip then simple majority is enough AFAIK.

Reply:
@dirvine I tried to search for n*n but couldn't find the reference, mind expanding? or just a link?